postgres-external-storage/src/test/java/com/netflix/conductor/postgres/storage/PostgresPayloadStorageTest.java

```diff
@@ -91,10 +107,44 @@ public void testReadInputStreamFromDb() throws IOException, SQLException {
         stmt.setString(1, key);
         stmt.setBinaryStream(2, inputData, inputData.available());
         stmt.executeUpdate();
+    }
 
-        assertEquals(
-                inputString,
-                new String(executionPostgres.download(key).readAllBytes(), StandardCharsets.UTF_8));
+    @Test
+    public void testMultithreadDownload()
+            throws ExecutionException, InterruptedException, SQLException, IOException {
+        insertData();
+        int numberOfThread = 12;
+        int taskInThread = 100;
+        ArrayList<CompletableFuture<?>> completableFutures = new ArrayList<>();
+        Executor executor = Executors.newFixedThreadPool(numberOfThread);
+        IntStream.range(0, numberOfThread * taskInThread)
+                .forEach(
+                        i -> {
```

Comment:
This is pretty hard to read beacause:

- lambda in lambda is not well readable, could you move the internal lambda into a method ?
- formatting, This is set to line width of 80 ? I think making this 120 chars would make it more readable

Reply:
Same for other tests